### ISSUE TITLE
Masses for docking ports are too high

### ISSUE THREAD

**Issue by ChrisAdderley** [ChrisAdderley/NearFutureConstruction#2]
Now that docking port strength is no longer related to mass, there is no point in the docking ports being as heavy as they are. They should be reduced to reasonable levels.

**Comment by Vladimir-csp** [comment]
here is my take on this, with rationales:

```
// Octo-Girder Docking Connector
@PART[trusslrg-docking-125]
{
    // 3x Clamp-O-Tron, has more stuff around.
    @mass = 0.15

    // restrict to 90 degree positions
    //@MODULE[ModuleDockingNode]{
    //  captureMinRollDot = 0.99995
    //  snapRotation = true
    //  snapOffset = 90
    //}
}

// Octo-Girder Heavy Docking Connector
@PART[trusslrg-docking-25]
{
    // 1.5x Clamp-O-Tron Sr.
    // It is less fatty and has no CLS. High in tech tree.
    // Should be same or lighter, but BALANCE!
    @mass = 0.3

    // restrict to 90 degree positions
    //@MODULE[ModuleDockingNode]{
    //  captureMinRollDot = 0.99995
    //  snapRotation = true
    //  snapOffset = 90
    //}
}

// Clamp-O-Tron Docking Module
@PART[crewtube-docking-125]
{
    // ~ PTD-02 / 3 + Clamp-O-Tron
    @mass = 0.15
}

// Clamp-O-Tron Sr. Docking Module
@PART[crewtube-docking-25]
{
    // ~ PPD-A2 / 3 + Clamp-O-Tron Sr.
    @mass = 0.6
}

// Grip-O-Tron
@PART[docking-25]
{
    // Far less fatty than Clamp-O-Tron Sr, has no CLS, High in tech tree.
    @mass = 0.15

    // Is that guide petals on the model? If so, the model should be rotated 60 degrees.
    // restrict to 120 degree positions
    //@MODULE[ModuleDockingNode]{
    //  captureMinRollDot = 0.99995
    //  snapRotation = true
    //  snapOffset = 120
    //}
}
```

**Comment by ChrisAdderley** [comment]
Resolved in 5eeb3c22dd5e729608896e6c8ce4723b967f855a. If you could log an issue in SSPX repo for the docking modules, that would be great.
